ui/public/locales/pt_BR.json

```diff
@@ -1,879 +1,1143 @@
-"label.associatednetworkname": "Nome da Rede",
+"label.associatednetworkid": "ID de rede associado",
+"label.associatednetworkname": "Nome da rede",
+"label.asyncbackup": "Backup ass\u00edncrono",
 "label.author.email": "E-mail do autor",
 "label.author.name": "Nome do autor",
-"label.autoscale": "Escalonamento Autom\u00e1tico",
-"label.autoscale.configuration.wizard": "Assistente de configura\u00e7\u00e3o de AutoScale",
-"label.availability": "Availability",
-"label.availabilityzone": "availabilityZone",
+"label.auto.assign.diskoffering.disk.size": "Atribuir automaticamente a oferta correspondente ao tamanho do disco",
+"label.auto.assign.random.ip": "Atribuir automaticamente um enderec\u0327o de IP aleat\u00f3rio",
+"label.autoscale": "Escalonamento autom\u00e1tico",
+"label.autoscale.configuration.wizard": "Assistente de configura\u00e7\u00e3o de escala autom\u00e1tica",
+"label.autoscalingenabled": "Escalonando automaticamente",
+"label.availability": "Disponibilidade",
+"label.availabilityzone": "Disponibilidade da zona",
 "label.available": "Dispon\u00edvel",
-"label.available.public.ips": "IP P\u00fablico Dispon\u00edvel",
+"label.available.public.ips": "Enderec\u0327os de IP p\u00fablicos dispon\u00edveis",
 "label.back": "Voltar",
+"label.backup": "Backups",
+"label.backup.attach.restore": "Restaurar e anexar volume de backup",
+"label.backup.offering.assign": "Atribuir VM a oferta de backup",
+"label.backup.offering.remove": "Remover VM de oferta de backup",
+"label.backup.offerings": "Ofertas de backup",
+"label.backup.restore": "Restaurar backup da VM",
+"label.backupofferingid": "Oferta de backup",
 "label.balance": "Saldo",
-"label.bandwidth": "Bandwidth",
-"label.baremetal.dhcp.devices": "Dispositivos DHCP Baremetal",
-"label.baremetal.dhcp.provider": "Provedor DHCP Baremetal",
-"label.baremetal.pxe.device": "Adiciona Dispositivo PXE Baremetal",
-"label.baremetal.pxe.devices": "Dispositivo PXE Baremetal",
-"label.baremetal.pxe.provider": "Provedor PXE Baremetal",
-"label.baremetal.rack.configuration": "Configura\u00e7\u00e3o do Rack de Baremetal",
+"label.bandwidth": "Largura de banda",
+"label.baremetal.dhcp.devices": "Dispositivos DHCP baremetal",
+"label.baremetal.dhcp.provider": "Provedor DHCP baremetal",
+"label.baremetal.pxe.device": "Adiciona dispositivo PXE baremetal",
+"label.baremetal.pxe.devices": "Dispositivo PXE baremetal",
+"label.baremetal.pxe.provider": "Provedor PXE baremetal",
+"label.baremetal.rack.configuration": "Configura\u00e7\u00e3o do rack de baremetal",
 "label.baremetalcpu": "CPU (em MHz)",
-"label.baremetalcpucores": "# de Core CPU",
-"label.baremetalmac": "Host MAC",
+"label.baremetalcpucores": "# de n\u00facleos da CPU",
+"label.baremetalmac": "MAC do host",
 "label.baremetalmemory": "Mem\u00f3ria (em MB)",
+"label.based.on": "Baseado em",
+"label.based.on.role.id.or.type": "Cria uma fun\u00e7\u00e3o baseado no ID da fun\u00e7\u00e3o ou no seu tipo",
```

Comment:
I would go with `Criar` here. But I am not really sure on the context, "Cria" could as well fit.
What do you think @GutoVeronezi?

Reply:
In the English version, it is `Creates a role based on either role id or type`, so it makes sense to translate to `Criar` in this case.